frontend/src/components/SQLReview/SQLReviewInfo.vue

```diff
@@ -1,26 +1,76 @@
 <template>
   <div class="space-y-6">
-    <div v-if="selectedEnvironment">
+    <div>
       <label class="textlabel">
-        {{ $t("sql-review.create.basic-info.environments") }}
+        {{ $t("sql-review.create.basic-info.attach-resource") }}
         <span style="color: red">*</span>
       </label>
       <p class="mt-1 textinfolabel">
-        {{ $t("sql-review.create.basic-info.environments-label") }}
+        {{ $t("sql-review.create.basic-info.attach-resource-label") }}
       </p>
-      <BBAttention
-        v-if="availableEnvironmentList.length === 0"
+      <NRadioGroup
+        v-model:value="attachResourceType"
+        :disabled="!allowChangeAttachedResource"
+        class="space-x-2 mt-2"
+      >
+        <NRadio value="environment">{{ $t("common.environment") }}</NRadio>
+        <NRadio value="project">{{ $t("common.project") }}</NRadio>
+        <NRadio
+          v-if="isDev() && attachResourceType !== 'database'"
```

Comment:
Since there isn't recent plan for database level, let's just remove it for now?

Reply:
Done